testing/ember-app/tests/utils/function-resource/rendering-test.ts

```diff
@@ -488,4 +492,44 @@ module('Utils | resource | rendering', function (hooks) {
       ]);
     });
   });
+
+  module('with owner', function (hooks) {
+    class TestService extends Service {
+      @tracked count = 1;
+    }
+
+    class Test {
+      // @use is required if a primitive is returned
+      @use data = resource(() => {
+        const test = (getOwner(this) as Owner).lookup('service:test') as TestService;
```

Comment:
same here -- the `getOwner` call is running off of the `Test`, rather than having anything to do with the resource